tests/Unit/PointwiseFunctions/GeneralRelativity/Test_ComputeSpacetimeQuantities.cpp

```diff
@@ -432,29 +117,58 @@ void test_compute_spatial_metric_lapse_shift(const T& used_for_size) {
 
 SPECTRE_TEST_CASE("Unit.PointwiseFunctions.GeneralRelativity.SpacetimeDecomp",
                   "[PointwiseFunctions][Unit]") {
-  const DataVector dv(2);
-  test_compute_1d_spacetime_metric(dv);
-  test_compute_2d_spacetime_metric(dv);
-  test_compute_3d_spacetime_metric(dv);
-  test_compute_1d_inverse_spacetime_metric(dv);
-  test_compute_2d_inverse_spacetime_metric(dv);
-  test_compute_3d_inverse_spacetime_metric(dv);
-  test_compute_1d_derivatives_spacetime_metric(dv);
-  test_compute_2d_derivatives_spacetime_metric(dv);
-  test_compute_3d_derivatives_spacetime_metric(dv);
+  pypp::SetupLocalPythonEnvironment local_python_env(
+      "PointwiseFunctions/GeneralRelativity/");
+
+  const DataVector dv(5);
+  const double d = std::numeric_limits<double>::signaling_NaN();
+
+  test_compute_spacetime_metric<1>(dv);
```

Comment:
What had we concluded about a macro for all these repetitive calls?

Reply:
we asked @carmaza to add them in #712

Reply:
Yes, and after looking at this PR some days ago, I decided to work on a more general macro handling functions with arbitrary template parameters. I'm still trying to make it work though...